.env.development

```diff
@@ -1,5 +1,5 @@
 NODE_ENV='development'
-PORT=8080
+PORT=8088
 ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
 BASE_URL='localhost:8080'
```

Comment:
Don't we also need to change port here?

Reply:
@xitij2000 probably, that was changed for test purposes because `8080` port already bound in your system.